src/test/java/service/implementation/CsvDataReaderImplTest.java

```diff
@@ -0,0 +1,72 @@
+package service.implementation;
+
+import static org.junit.Assert.assertEquals;
+
+import db.FruitStorage;
+import exception.InvalidDataFormatException;
+import java.util.ArrayList;
+import java.util.List;
+import model.Fruit;
+import model.FruitTransactionDto;
+import model.Operation;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Test;
+import service.DataReader;
+
+public class CsvDataReaderImplTest {
+    public static final String VALID_INPUT = "src/test/resources/valid_input.csv";
+    public static final String INVALID_AMOUNT_NAN = "src/test/resources/invalid_amount_NaN.csv";
+    public static final String INVALID_FORMAT = "src/test/resources/invalid_format.csv";
+    public static final String INVALID_OPERATION_IS_NUMBER = "src/test/resources/"
+            + "invalid_operation_isNumber.csv";
+
+    private static List<FruitTransactionDto> expectedFruits;
+    private static DataReader dataReader;
+
+    @Before
```

Comment:
Do you need to create these data before each test method?

Reply:
I thought it is good to initialise Object in the `Before` method, I see that it does not make sense workflow wise. Thank you